arch/arm/boot/dts/imx6ul-ts7180.dts

```diff
@@ -0,0 +1,1020 @@
+		ethphy0: ethernet-phy@2 {
+			clocks = <&clks IMX6UL_CLK_ENET_REF>;
+			clock-names = "rmii-ref";
+			compatible = "ethernet-phy-ieee802.3-c22";
+			reg = <2>;
+		};
+
+		ethphy1: ethernet-phy@1 {
+			clocks = <&clks IMX6UL_CLK_ENET_REF>;
+			clock-names = "rmii-ref";
+			compatible = "ethernet-phy-ieee802.3-c22";
+			reg = <1>;
+		};
+	};
+};
+
+&gpio1 {
+	gpio-line-names =
+		"BOOT_MODE_0",
+		"NA",
+		"I2C_1_CLK",
+		"I2C_1_DAT",
+		"ADC_1",
+		"ADC_2",
+		"ETH_MDIO",
+		"ETH_MDC",
+		"ADC_3",
+		"ADC_4",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"CONSOLE_TXD",
+		"CONSOLE_RXD",
+		"SPARE_1",
+		"EN_485",
+		"UART2_TXD",
+		"UART2_RXD",
+		"CAN_2_TXD",
+		"CAN2_RXD_3V",
+		"UART3_TXD",
+		"UART3_RXD",
+		"UART3_CTS#",
+		"UART3_RTS#",
+		"UART4_TXD",
+		"UART4_RXD",
+		"UART5_TXD",
+		"UART5_RXD";
+};
+
+&gpio2 {
+	gpio-line-names =
+		"ENET1_RX_DATA0",
+		"ENET1_RX_DATA1",
+		"ENET1_RX_EN",
+		"ENET1_TX_DATA0",
+		"ENET1_TX_DATA1",
+		"ENET1_TX_EN",
+		"ENET1_TX_CLK",
+		"ENET1_RX_ER",
+		"ENET2_RX_DATA0",
+		"ENET2_RX_DATA1",
+		"ENET2_RX_EN",
+		"ENET2_TX_DATA0",
+		"ENET2_TX_DATA1",
+		"ENET2_TX_EN",
+		"ENET2_TX_CLK",
+		"ENET2_RX_ER",
+		"SD_CMD",
+		"SD_CLK",
+		"SD_D0",
+		"SD_D1",
+		"SD_D2",
+		"SD_D3",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpio3 {
+	gpio-line-names =
+		"HD12_SPI_CS",
+		"JTAG_FPGA_TCK",
+		"JTAG_FPGA_TMS",
+		"JTAG_FPGA_TDI",
+		"WDOG#",
+		"I2C_3_DAT",
+		"I2C_3_CLK",
+		"NA",
+		"HD12_I2C_CLK",
+		"HD12_I2C_DAT",
+		"HD12_DIG_INPUT",
+		"NO_CHRG_JMP#",
+		"EN_NIM_USB#",
+		"CAN_1_TXD",
+		"CAN1_RXD_3V",
+		"XBEE_CTS#",
+		"U_BOOT_JMP#",
+		"NA",
+		"PUSH_SW_CPU#",
+		"NIMBEL_PWR_ON",
+		"NA",
+		"UART7_TXD",
+		"UART7_RXD",
+		"ID4",
+		"JTAG_FPGA_TDO",
+		"UART8_TXD",
+		"UART8_RXD",
+		"ID1",
+		"ETH_PHY_RESET#",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpio4 {
+	gpio-line-names =
+		"EMMC_CLK",
+		"EMMC_CMD",
+		"EMMC_D0",
+		"EMMC_D1",
+		"EMMC_D2",
+		"EMMC_D3",
+		"SPI_4_CLK",
+		"SPI_4_MOSI",
+		"SPI_4_MISO",
+		"SPI_4_CS#",
+		"MAG_N_IRQ",
+		"FPGA_RESET#",
+		"SPI_3_FPGA_CS#",
+		"SPI_3_CLK",
+		"SPI_3_MOSI",
+		"SPI_3_MISO",
+		"PWM_5",
+		"UART6_TXD",
+		"UART6_RXD",
+		"ID5",
+		"GYRO_INT",
+		"6UL_FORCE_5V_ON",
+		"EN_EMMC_3.3V#",
+		"EN_YEL_LED#",
+		"EN_RED_LED#",
+		"EN_GRN_LED#",
+		"EN_BLU_LED",
+		"FRAM_SPI_CS#",
+		"SD_VSEL_1.8V",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpio5 {
+	gpio-line-names =
+		"POWER_FAIL",
+		"GPS_PPS_OUT",
+		"NA",
+		"GPIO_DVFS",
+		"NA",
+		"SILAB_C2_CLK",
+		"SILAB_C2_DATA",
+		"SILAB_C2_RESET",
+		"SPARE_4",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA",
+		"NA";
+};
+
+&gpc {
+	fsl,cpu_pupscr_sw2iso = <0x1>;
+	fsl,cpu_pupscr_sw = <0x0>;
+	fsl,cpu_pdnscr_iso2sw = <0x1>;
+	fsl,cpu_pdnscr_iso = <0x1>;
+	fsl,ldo-bypass = <0>; /* DCDC, ldo-enable */
+};
```

Comment:
Check that the kernel supports these. I think these are only used in NXP specific trees and vanilla kernel doesn't use them.